vm/integer_test.go

```diff
@@ -27,6 +27,19 @@ func TestEvalInteger(t *testing.T) {
 		{`4 > 6`, false},
 		{`-5 < -4`, true},
 		{`100 < 81`, false},
+		{`5 ** 4`, 625},
+		{`25 / 5`, 5},
+		{`25 > 5`, true},
+		{`4 > 6`, false},
+		{`4 >= 4`, true},
+		{`2 >= 5`, false},
+		{`-5 < -4`, true},
+		{`100 < 81`, false},
+		{`10 <= 10`, true},
+		{`10 <= 0`, false},
+		{`10 <=> 0`, 1},
+		{`1 <=> 2`, -1},
+		{`4 <=> 4`, 0},
```

Comment:
Can you add more complicated arithmetic tests in here? Which should be a combination of different operators.